source/evp_fetch.c

```diff
@@ -101,10 +101,24 @@ void do_fetch(size_t num)
     OSSL_TIME start, end;
     size_t i, j;
     const char *fetch_alg = NULL;
+    size_t x_times;
+
+    /*
+     * Make sure all the fetch entries are called and not only the first x
+     */
+    x_times = num_calls / threadcount;
+    if (x_times % ARRAY_SIZE(fetch_entries) > 0) /* round up */
+        x_times += ARRAY_SIZE(fetch_entries) - x_times % ARRAY_SIZE(fetch_entries);
+
+    printf("num_calls: %zu, threadcount: %d, x_times: %zu, array size fetch_entries: %zu\n",
```

Comment:
I'm not sure about printf() because it may break the script which collects data for grafana. I think the tool must output just result when running with terse option `-t`